reporter/src/main/kotlin/reporters/ReportTableModelMapper.kt

```diff
@@ -24,7 +24,10 @@ import com.here.ort.model.OrtIssue
 import com.here.ort.model.OrtResult
 import com.here.ort.model.RemoteArtifact
 import com.here.ort.model.RuleViolation
+import com.here.ort.model.Scope
```

Comment:
> not worth a convenience function on the model level as long as there is no re-use.

On the other hand, *if* somebody wanted to re-use this in the future, it's now harder to find. This is just a remark; I'm not saying you should drop this change.

Reply:
In hindsight it would have been better to first inline the function in the model mapper as you do in a later commit and then delete it afterwards because it is unused. But the argument in the commit message is not convincing, especially since you add another function in one of the next commits which is also only used in a single place.
But anyway, since the outcome is the same keep it as is.

Reply:
I guess with the function used in a single place you are referring to [1], which actually has two callers - not one.

[1] https://github.com/heremaps/oss-review-toolkit/pull/2231/commits/7179f5de15956061bb9aef24bed6e9f311316f61

Reply:
Only if you count the other `findScopeExcludes` as a caller, but that's obviously not what this argument is about.

Reply:
I'm not following the argumentation, but anyhow let's just resolve the discussion here as there seems to be no mandatory change request in here.